playbooks/keyholders/EXISTING_SIGNER.md

```diff
@@ -101,9 +101,10 @@ This will setup your clone and build the TUF binary to use for metadata generati
 
 ### Signing
 
-When prompted in the Slack channel, begin signing root and targets metadata. Run:
+When prompted in the Slack channel, begin signing root and targets metadata. If the ceremony started on a prior date, you should add a repository reference by setting `REPO`. Otherwise, for same-day signing, you may omit to default to the current date. Run:
```

Comment:
```suggestion
When prompted in the Slack channel, begin signing root and targets metadata. If the ceremony started on a prior date (or the date in your timezone does not match the date in the orchestrator's timezone), you should add a repository reference by setting `REPO`. Otherwise, for same-day signing, you may omit to default to the current date. Run:
```

Reply:
Hahah thank you! This was almost always going to be true for you!

Reply:
You'd have thought I'd remember the correct format for `REPO` by now, but had to figure it out again be reading the scripts today. Glad to have this written down!